tests/test_api_v1.py

```diff
@@ -1,6 +1,8 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 from unittest import mock
 
+import os
+import tempfile
 import kombu.exceptions
 import pytest
```

Comment:
Imports are ordered in the order of standard library, third-party, and then local.

```suggestion
import os
import tempfile
from unittest import mock

import kombu.exceptions
import pytest
```